src/layers/conv.jl

```diff
@@ -480,6 +480,63 @@ end
 outdims(l::CrossCor, isize) =
   output_size(DenseConvDims(_paddims(isize, size(l.weight)), size(l.weight); stride = l.stride, padding = l.pad, dilation = l.dilation))
 
+"""
+    AdaptiveMaxPool(out)
+
+Adaptive Max pooling layer. `out` is the size of the dimension of the output.
```

Comment:
Minor issues: Change "Adaptive Max pooling layer" to "Adaptive max pooling layer."

Can you add a sentence mentioning that the output size is fixed for arbitrary input sizes?